config/deploy/production.rb

```diff
@@ -1,2 +1,4 @@
 server 'trase.earth', user: 'ubuntu', roles: %w{web app db}, primary: true
 set :ssh_options, forward_agent: true
+
+after 'deploy:finishing', 'sitemap:refresh'
```

Comment:
Can we add the "no ping" version to staging temporarily, so we can test in real life that the sitemap is stored where it needs to be and is publicly accessible. This might require some config changes in the web server, and it would be good to test all of that fully on staging.